index.html

```diff
@@ -126,7 +127,35 @@
 				</p>
 			</div> <!-- cd-faq-content -->
 		</li>
-	</ul> <!-- cd-faq-group -->
+	</ul> 
+        <ul id="frameworki" class="cd-faq-group">
+                <li class="cd-faq-title"><h2>Frameworki</h2></li>
+                <li>
+                        <a class="cd-faq-trigger" href="#0">Laravel</a>
+                        <div class="cd-faq-content">
+                                <p>Eloquent (dostęp do bazy danych):</p>
+                                <ol>
+                                        <li>Dostępne są dwa query buildery: użycie <a href="https://github.com/laravel/framework/blob/v5.0.32/src/Illuminate/Database/Eloquent/Builder.php">pierwszego z nich</a> powoduje zwrócenie obiektów Active Record, natomiast <a href="https://github.com/laravel/framework/blob/v5.0.32/src/Illuminate/Database/Query/Builder.php">drugi</a> zwraca tablice.</li>
```

Comment:
https://github.com/laravel/framework/blob/v5.0.32/src/Illuminate/Database/Eloquent/Builder.php to nie jest query builder - to jest jakiś tam Eloquent builder. To jest https://github.com/laravel/framework/blob/v5.0.32/src/Illuminate/Database/Query/Builder.php query builder i on faktycznie nie powinien wykonywać query tylko je budować (tworzyć -> uzupełniać  -> zwracać). 

Co do Dotrine to nie mieszajmy w to DBAL'a. Te przypadki nie są w ogóle do siebie podobne. Encji nie zwraca repozytorium tylko hydrator wywołany na query podczas wykonywania getResults. Samo query natomiast można zbudować QueryBuilderem. Jednak niezależnie od tego jaką strategię hydracji wyników wybierzesz samo zapytanie do bazy się nie zmieni, hydracja wykonuje się na pobranych danych a nie na zapytaniu.